src/params/pointscatter.params

```diff
@@ -1,4 +1,5 @@
 Experiment ID                :pointscatter
+Data Representation          :thetamagpolar
```

Comment:
I think we can even make a couple parameter files, I'll make sure subsequent additions have different names than this one.